openedx/core/djangoapps/user_authn/docs/decisions/0001_simplify_ratelimiting.rst

```diff
@@ -0,0 +1,106 @@
+Simplify Login and Other Rate Limiting
```

Comment:
@robrap I made this more generic as you suggested because I think the advice is more generically applicable.   But I think in this case I should also move it to the top level decisions folder, do you agree?